x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -85,9 +84,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     return tableData;
   };
 
-  describe('endpoint list', function () {
-    targetTags(this, ['@ess', '@serverless']);
```

Comment:
Q: why don't we need tags anymore? Are we reading them from the `describe` text now?

Reply:
yes, we have moved tags to the beginning of description.